paddle/framework/lod_tensor.cc

```diff
@@ -152,6 +152,45 @@ void LoDTensor::ShrinkInLevel(size_t level, size_t elem_begin,
   ShareDataWith(Slice(begin, end));
 }
 
+void LoDTensor::SerializeToStream(
```

Comment:
I believe we already have a `SerializeToStream` interface.  Furthermore, SerializeToStream should be a global function. Tensor and LoDTensor only describe a math computing library.

Reply:
Yes, that `SerializeToStream ` only support `LoDTensor`, and I believe make `SerializeToStream `  as a global interface is a good choice as the discussion  just not.